### PR TITLE
Getting shrunk makes your voice small

### PR DESCRIPTION
**Pull request by dragomagol**
## About The Pull Request
image
image
image
image

## Why It's Good For The Game
I think it's funny. Plus I like what helium introduced with the small text, and wanted to add it to more places.

## Changelog
:cl: Tattle
add: Getting shrunk makes you talk small
/:cl:

### CODE REVIEW

**Comment by Jacquerel** [comment]
Good PR, although it'd be great if we could do it to TTS as well.

**Review by Jacquerel — APPROVED** [review]

File: code/datums/components/shrink.dm [hunk 1]
```diff
@@ -30,6 +31,10 @@
 	span_userdanger("Everything grows bigger!"))
 	QDEL_IN(src, shrink_time)
 
+/datum/component/shrink/proc/handle_shrunk_speech(mob/living/breather, list/speech_args)
+	SIGNAL_HANDLER
+	speech_args[SPEECH_SPANS] |= SPAN_HELIUM
```
Comment (ZephyrTFA) [inline code/datums/components/shrink.dm:36]: change the span name to SPAN_SMALL_TEXT to increase readability

**Comment by Jackal-boop** [comment]
could you please make this stack with helium

**Comment by dragomagol** [comment]
Out of scope

**Pull request merged**